src/qt/rpcconsole.cpp

```diff
@@ -1312,11 +1312,8 @@ void RPCConsole::unbanSelectedNode()
     {
         // Get currently selected ban address
         QString strNode = nodes.at(i).data().toString();
-        CSubNet possibleSubnet;
-
-        LookupSubNet(strNode.toStdString(), possibleSubnet);
-        if (possibleSubnet.IsValid() && m_node.unban(possibleSubnet))
-        {
+        const CSubNet possibleSubnet{LookupSubNet(strNode.toStdString())};
+        if (m_node.unban(possibleSubnet)) {
```

Comment:
The new code could call `unban()` with an invalid subnet whereas the previous one would not. `unban()` would call `m_banned.erase()` which would use `operator==(const CSubNet& a, const CSubNet& b)` which could match another invalid entry, I guess. Maybe that is ok, but to keep this change "non-functional", the new code should be:

```cpp
if (possibleSubnet.IsValid() && m_node.unban(possibleSubnet)) {
```

Reply:
Makes sense! Gonna address it!